### DISCUSSION TITLE
Unable to resolve host "************.supabase.co": No address associated with hostname

### DISCUSSION THREAD

**Discussion by galverlic** [supabase/supabase#20900] [Questions]
[0:] Error in GetHabits: Unable to resolve host "************.supabase.co": No address associated with hostname

I made no changes to my code, yesterday application worked fine, today I tried to run it and got this error in my output.

**Comment by GaryAustin1** [comment]
If you were using the direct database port and your internet/network does not support IPv6 then you needed to move to the new Supavisor connections.
https://github.com/orgs/supabase/discussions/17817

**Reply by galverlic** [reply]
I'm not connectind directly i'm using the API from supabase, the url from porject url and the project api key in my AppConfig and then passing it to my MauiProgram.cs
as  var url = AppConfig.SUPABASE_URL;
 var key = AppConfig.SUPABASE_KEY;
 builder.Services.AddSingleton(provider => new Supabase.Client(url, key));

**Reply by GaryAustin1** [reply]
Restart your instance.
If that does not work and you have not already put in a support request please do so.
The IP changes should not be impacting the REST API calls.

**Reply by trancethehuman** [reply]
I'm having the same issue as well (PostgREST, Auth and Storage affected). Can't even load dashboard @GaryAustin1 

Also tried restarting but still in the same state.

**Reply by trancethehuman** [reply]
![image](https://github.com/supabase/supabase/assets/16231195/6108eac3-90b5-43d7-b6aa-784ee2617534)

**Reply by GaryAustin1** [reply]
If it does not clear shortly then you need to contact support.

**Comment by galverlic** [comment]
Last night i refreshed and it started working normally, i don't know why..

**Comment by trancethehuman** [comment]
I reached out to support and they said it was an edge case for my project to be stuck in Restarting / pausing because of IPv4 -> IPv6. They resolved it in 24hrs.

**Comment by Throvn** [comment]
I was in China and had the same problems as @galverlic. 
My connection worked fine (when VPN was disabled) the day before but not the next day anymore.
Switching out the DATABASE_URL to use a connection pooler instead of the direct connection solved the issue for me.

Thanks @GaryAustin1
